.gitattributes

```diff
@@ -0,0 +1,9 @@
+/.gitattributes export-ignore
```

Comment:
what's wrong with using `.gitignore`?

Reply:
This file serves a different purpose: https://git-scm.com/docs/gitattributes#_export_ignore
TL;DR: It allows you to exclude files from the archive, thus excluding them when installing the package via composer.